RestaurantBusiness/Controllers/RestaurantApiController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using RestaurantBusiness.DatabaseConfiguration;
+using RestaurantBusiness.Models;
+using RestaurantBusiness.Repositories;
+using System.Threading.Tasks;
+
+namespace RestaurantBusiness.Controllers
+{
+    [Route("api/restaurants")]
+    [ApiController]
+    public class RestaurantApiController : ControllerBase
+    {
+        private readonly RestaurantRepository _restaurantRepository;
+
+        public RestaurantApiController(IOptions<CosmosDbSettings> options)
+        {
+            _restaurantRepository = new RestaurantRepository(options);
```

Comment:
You have injected the options through DI but you instantiate the repo manually. Would be better to inject it as well.